src/field.js

```diff
@@ -79,7 +79,12 @@ function computeFieldObject (data, template, options) {
   if (!metadata.fields && data.statistics) return computeFields(data.statistics[0], template, options).fields
   else if (!metadata.fields) return computeAggFieldObject(data, template, options)
 
-  const fields = metadata.fields.map(field => {
+  var outFields = options.outFields.split(",")
```

Comment:
For now, lets return all fields. It's going to be a bit more work to return no fields at all when outFields is empty as is the case with ArcGIS Server.